CompuCell3D/core/Demos/SteppableDemos/PolygonInitializer/Simulation/cellsort_2D.xml

```diff
@@ -0,0 +1,208 @@
+<CompuCell3D>
```

Comment:
Do you think we should add a comment at the top of each one like this?
``<!--See the documentation for more details: https://compucell3dreferencemanual.readthedocs.io/en/latest/polygon_initializer.html -->``
And, similarly, for tubes...
``<!--See the documentation for more details: https://compucell3dreferencemanual.readthedocs.io/en/latest/tube_initializer.html -->``